src/components/Comments/Comment.js

```diff
@@ -38,6 +46,7 @@ const Comment = ({
   onLongPress,
   updateVote
 }) => {
+  const refSheet = React.useRef();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `refSheet` is defined but it's not clear what it's used for. Please ensure that this ref is necessary and used appropriately in the component.

```diff
+  const refSheet = React.useRef(null);
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->